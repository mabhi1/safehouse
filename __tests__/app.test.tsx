import "@testing-library/jest-dom";
import Home from "@/app/page";
import { render, screen } from "@testing-library/react";

describe("Home Page", () => {
  it("renders page without any errors", async () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();

    const heading = screen.getByRole("heading", { level: 2 });
    expect(heading).toHaveTextContent("Safe House");
  });
});
