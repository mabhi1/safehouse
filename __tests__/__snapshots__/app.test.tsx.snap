// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders page without any errors 1`] = `
<div>
  <div>
    <div
      class="flex flex-col md:flex-row"
    >
      <img
        alt="Safe House"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="300"
        src="/hero.svg"
        style="color: transparent;"
        width="300"
      />
      <div
        class="flex flex-col justify-center p-5 gap-2 w-full md:w-1/2"
      >
        <h2
          class="text-2xl"
        >
          Safe House
        </h2>
        <div>
          Your personal Secure storage for passwords, banks, notes and more. All the data for your cards and passwords are stored in an encrypted form.
        </div>
      </div>
    </div>
  </div>
</div>
`;
